cobalt/site/docs/reference/starboard/gn-configuration.md

```diff
@@ -35,6 +35,7 @@ title: "Starboard: configuration.gni Reference Guide"
 | **`sb_filter_based_player`**<br><br> Used to indicate that the player is filter based.<br><br>The default value is `true`. |
 | **`sb_is_evergreen`**<br><br> Whether this is an Evergreen build.<br><br>The default value is `false`. |
 | **`sb_is_evergreen_compatible`**<br><br> Whether this is an Evergreen compatible platform. A compatible platform can run the elf_loader and launch the Evergreen build.<br><br>The default value is `false`. |
+| **`sb_is_modular`**<br><br> Whether this is a modular build.<br><br>The default value is `false`. |
```

Comment:
@niranjanyardi `sb_is_modular` is still set in `base_configuration.gni`—it should not be configurable by the platform, so this should not be the case. We should remove it from there.

@oxve Can you manually revert this addition for now?

Reply:
I can manually revert it. This list is generated by gn in the site generation script so it will get added next time we run it.

Is that what you expect or is this an issue with our GN configuration?

Reply:
andrew is right - that value is in starboard/build/config/base_configuration.gni ,

Reply:
Indeed, that is where the script looks for variables that can be configured.

Reply:
This is what we expect, it was just left there by accident and needs to be removed

Reply:
Removed